.gitignore

```diff
@@ -1,35 +1,114 @@
+# Created by https://www.gitignore.io/api/python
```

Comment:
You could submit some of these options to [github's gitignore repository](https://github.com/github/gitignore)

Reply:
@peterfpeterson do you mean these lines at the end?
```
# NeXus and HDF5 files
*.nxs
*.h5
*.hdf5
```